windows/src/engine/keyman32-ver/calldll.cpp

```diff
@@ -30,6 +30,12 @@
 
 typedef BOOL (WINAPI *InitDllFunction)(PSTR name);
 
+#ifdef _WIN64
+  const char keyman_engine_arch[] = "keyman64.dll";
+#else
+  const char keyman_engine_arch[] = "keyman32.dll";
+#endif
+
```

Comment:
@marc Not sure if this is what you had in mind or something in globals.h ?

Reply:
Yes, this works. Let's use `wchar_t` instead of `char`.